.cbmc-batch/source/openssl/evp_override.c

```diff
@@ -624,12 +629,17 @@ int EVP_DecryptUpdate(EVP_CIPHER_CTX *ctx, unsigned char *out, int *outl, const
     __CPROVER_assume(out_size >= 0);
     if (ctx->cipher) {
```

Comment:
Not necessary for this PR, but using `AWS_IMPLIES` might make the logic here more explicit.

Reply:
Agreed. We can implement these improvements on a second round of reviews.

Reply:
Add a baclkog item so we don't lose track of this

Reply:
https://github.com/aws/aws-encryption-sdk-c/issues/525 done.